.gitlab-ci.yml

```diff
@@ -86,33 +73,38 @@ test-linux-stable:                 &test
     variables:
       - $DEPLOY_TAG
   before_script:
-   - test -d ${CARGO_HOME} -a -d ./target &&
-     echo "build cache size:" &&
-     du -h --max-depth=2 ${CARGO_HOME} ./target
-   - ./scripts/build.sh
+    - sccache -s
+    - ./scripts/build.sh
   script:
     - time cargo test --all --release --verbose --locked
+    - sccache -s
 
-
-
-
+check-web-wasm:
+  stage:                           test
+  image:                           tomaka/cargo-web:latest
+  allow_failure: true
+  script:
+    # WASM support is in progress. As more and more crates support WASM, we
+    # should add entries here.
+    - time cargo web build -p substrate-network-libp2p
+    - time cargo web build -p substrate-panic-handler
+    - time cargo web build -p substrate-peerset
+  only:
+    - master
+    - /^[0-9]+$/
+  except:
+    - v1.0
```

Comment:
I guess the only statements exclude the excepts already.

Reply:
it doesn't work at all since gitlab doesn't know our branches. I DMed you.